### PR TITLE
Deploy application with nginx and application_python

### PR DESCRIPTION
**Pull request by tschuy**

### CODE REVIEW

**Comment by tschuy** [comment]
This also fails to make the supervisor service ``whats_fresh`` start on machine reboot.

I'll change that .kitchen.cloud.yml openstack_tenant var, along with the getting the default suite to work.  What I'm most likely going to do is add the postgresql::default to the default runlist. Any objections to this?

**Comment by iankronquist** [comment]
@tschuy are you still working on this?

**Comment by tschuy** [comment]
This commit fixes the default converge, and the openstack variable. 

Upon reboot, supervisor itself is running, but whats-fresh is not:

```
[centos@defaultcentos65-tschuy-aqua-m61fx8i ~]$ ps aux | grep super
root        18  0.0  0.0      0     0 ?        S    20:29   0:00 [sync_supers]
root      1435  0.0  0.5 209636 10572 ?        Ss   20:29   0:00 /usr/bin/python2.7 /usr/bin/supervisord -c /etc/supervisord.conf
centos    1502  0.0  0.0 103212   600 pts/0    R+   20:30   0:00 grep super
[centos@defaultcentos65-tschuy-aqua-m61fx8i ~]$
```

**Comment by tschuy** [comment]
Turns out there's an attribute for that. I've set it.

Is there anything left needing to be done at this point?

**Comment by ramereth** [comment]
@tschuy mind bumping CentOS to 6.6 on the test kitchen configs and verify it works?

File: test/integration/develop/serverspec/postgresql_running_spec.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 include Serverspec::Helper::Exec
 include Serverspec::Helper::DetectOS
```
Comment (ramereth) [inline test/integration/develop/serverspec/postgresql_running_spec.rb]: The two above lines need to be converted over to the newer style:

``` ruby
set :backend, :exec
```

File: test/integration/develop/serverspec/postgresql_running_spec.rb [hunk 2]
```diff
@@ -12,7 +12,7 @@
   end
 end
 
-describe "create Postgis extension to database and create table" do
+describe 'create Postgis extension to database and create table' do
   describe command("runuser -l postgres -c 'psql whats_fresh -c \"\\dx\"'") do
     it { should return_stdout /(.*?)postgis(.*)/ }
```
Comment (ramereth) [inline test/integration/develop/serverspec/postgresql_running_spec.rb]: This is no longer working, the updated method for matching stdout text is: http://serverspec.org/resource_types.html#command

**Comment by tschuy** [comment]
I've bumped to 6.6, which is working, and udpated the serverspec tests.

**Comment by ramereth** [comment]
I'm OK :+1: this as-is and do other fixes and improvements later.

**Pull request merged**